layouts/partials/footer.html

```diff
@@ -28,15 +28,25 @@ <h3>{{ .title }}</h3>
       {{- end }}
         <div class="footer-actions">
             <p>Sign up for the latest NumPy news, resources, and more!</p>
-            <div class="sign-up-container">
+            <!-- Begin Mailchimp -->
+            <form action="https://numpy.us4.list-manage.com/subscribe/post?u=5ddd0d1d6e807900a8212481a&amp;id=287fa4253c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate sign-up-container" target="_blank" novalidate>
                 <i class="fal fa-envelope sign-up-image"></i>
-                <input aria-label="Input for email" id="email" class="sign-up-input" onkeypress="if (event.which === 13 || event.keyCode === 13 || event.key === 'Enter') enterEmail()" />
+                <input type="email" value="" name="EMAIL" class="required email sign-up-input" id="mce-EMAIL" aria-label="Input for email" onkeypress="if (event.which === 13 || event.keyCode === 13 || event.key === 'Enter') sendThankYou()" />
                 <div class="submission-instructions">Press Enter</div>
-                <button class="signup-button" onclick="enterEmail()" aria-label="Submit">
+                <button class="signup-button" onclick="sendThankYou()" aria-label="Submit">
                     <i class="fal fa-paper-plane"></i>
                 </button>
-            </div>
+                <div id="mce-responses" class="clear">
+                    <div class="response" id="mce-error-response" style="display:none"></div>
+                    <div class="response" id="mce-success-response" style="display:none"></div>
+                </div>
+                <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_5ddd0d1d6e807900a8212481a_287fa4253c" tabindex="-1" value=""></div>
+                <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" style="display: none;"></div>
+            </form>
             <div class="thank-you">Thank you! &#127881;</div>
+            <!--End MailChimp -->
+
```

Comment:
This is all boilerplate from MailChimp. I adapted our current HTML to work with this form.

Reply:
LGTM. Thank you, @joelachance!